### PR TITLE
Clean code

### PR DESCRIPTION
**Pull request by GraysonChiang**
### Description

Just modify some code, and make it Cleaner more...

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/magento2?pullRequest=16841) <br/>All committers have signed the CLA.

**Comment by magento-engcom-team** [comment]
Hi @GraysonChiang. Thank you for your contribution 
Here is some useful tips how you can test your changes using Magento test environment.
Add the comment under your pull request to deploy test or vanilla Magento instance:
- `@magento-engcom-team give me test instance` - deploy test instance based on PR changes
- `@magento-engcom-team give me {$VERSION} instance` - deploy vanilla Magento instance

For more details, please, review the [Magento Contributor Assistant](https://github.com/magento/magento2/wiki/Magento-Contributor-Assistant) documentation

**Review by ihor-sviziev — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @ihor-sviziev, thank you for the review.
[ENGCOM-2366](https://jira.corp.magento.com/browse/ENGCOM-2366) has been created to process this Pull Request

**Comment by magento-engcom-team** [comment]
@GraysonChiang thank you for contributing. Please accept Community Contributors team invitation [here](https://github.com/orgs/magento/invitation) to gain extended permissions for this repository.

File: app/code/Magento/Catalog/Helper/Image.php [hunk 1]
```diff
@@ -859,7 +859,7 @@ public function getFrame()
      */
     protected function getAttribute($name)
     {
-        return isset($this->attributes[$name]) ? $this->attributes[$name] : null;
+        return $this->attributes[$name]?? : null;
```
Comment (ihor-sviziev) [inline app/code/Magento/Catalog/Helper/Image.php]: 1. Please add space there before `??`
2. Looks like there should not be `:`

**Review by ihor-sviziev — CHANGES_REQUESTED** [review]

**Comment by GraysonChiang** [comment]
Sor, I fixed it and commit that again.

File: app/code/Magento/Backend/Model/Menu/Builder.php [hunk 2]
```diff
@@ -102,6 +102,6 @@ public function getResult(\Magento\Backend\Model\Menu $menu)
      */
     protected function _getParam($params, $paramName, $defaultValue = null)
     {
-        return isset($params[$paramName]) ? $params[$paramName] : $defaultValue;
+        return $params[$paramName]?? $defaultValue;
```
Comment (ihor-sviziev) [inline app/code/Magento/Backend/Model/Menu/Builder.php]: Please add space before `??`

File: app/code/Magento/BundleImportExport/Model/Export/RowCustomizer.php [hunk 3]
```diff
@@ -321,7 +321,7 @@ function ($title, $storeName) {
      */
     protected function getTypeValue($type)
     {
-        return isset($this->typeMapping[$type]) ? $this->typeMapping[$type] : self::VALUE_DYNAMIC;
+        return $this->typeMapping[$type]??  self::VALUE_DYNAMIC;
```
Comment (ihor-sviziev) [inline app/code/Magento/BundleImportExport/Model/Export/RowCustomizer.php]: 1. Please add space before `??`
2. Remove 2nd space after `??`

File: app/code/Magento/Catalog/Helper/Output.php [hunk 4]
```diff
@@ -105,7 +105,7 @@ public function addHandler($method, $handler)
     public function getHandlers($method)
     {
         $method = strtolower($method);
-        return isset($this->_handlers[$method]) ? $this->_handlers[$method] : [];
+        return $this->_handlers[$method] ?? : [];
```
Comment (ihor-sviziev) [inline app/code/Magento/Catalog/Helper/Output.php]: Please remove `:` (and space hear it)

File: app/code/Magento/Catalog/Model/Product/Option.php [hunk 5]
```diff
@@ -312,7 +312,7 @@ public function getGroupByType($type = null)
             self::OPTION_TYPE_TIME => self::OPTION_GROUP_DATE,
         ];
 
-        return isset($optionGroupsToTypes[$type]) ? $optionGroupsToTypes[$type] : '';
+        return $optionGroupsToTypes[$type] ?? : '';
```
Comment (ihor-sviziev) [inline app/code/Magento/Catalog/Model/Product/Option.php]: please remove `:` (and space near it)

File: app/code/Magento/Catalog/Model/ResourceModel/Product/Collection.php [hunk 6]
```diff
@@ -931,7 +931,7 @@ private function mapConditionType($conditionType)
             'eq' => 'in',
             'neq' => 'nin'
         ];
-        return isset($conditionsMap[$conditionType]) ? $conditionsMap[$conditionType] : $conditionType;
+        return $conditionsMap[$conditionType] ??  $conditionType;
```
Comment (ihor-sviziev) [inline app/code/Magento/Catalog/Model/ResourceModel/Product/Collection.php]: please remove 2nd space after `??`

File: app/code/Magento/Catalog/Ui/Component/Listing/Columns.php [hunk 7]
```diff
@@ -80,6 +80,6 @@ public function prepare()
      */
     protected function getFilterType($frontendInput)
     {
-        return isset($this->filterMap[$frontendInput]) ? $this->filterMap[$frontendInput] : $this->filterMap['default'];
+        return $this->filterMap[$frontendInput] ??  $this->filterMap['default'];
```
Comment (ihor-sviziev) [inline app/code/Magento/Catalog/Ui/Component/Listing/Columns.php]: please remove 2nd space after `??`

File: lib/internal/Magento/Framework/DataObject/Copy/Config.php [hunk 8]
```diff
@@ -44,6 +44,6 @@ public function getFieldset($name, $root = 'global')
         if (empty($fieldsets)) {
             return null;
         }
-        return isset($fieldsets[$name]) ? $fieldsets[$name] : null;
+        return $fieldsets[$name] ??  null;
```
Comment (ihor-sviziev) [inline lib/internal/Magento/Framework/DataObject/Copy/Config.php]: Please remove 2nd space after `??`

File: lib/internal/Magento/Framework/View/Page/Config/Structure.php [hunk 9]
```diff
@@ -251,6 +251,6 @@ public function populateWithArray(array $data)
      */
     private function getArrayValueByKey($key, array $array)
     {
-        return isset($array[$key]) ? $array[$key] : [];
+        return $array[$key] ??  [];
```
Comment (ihor-sviziev) [inline lib/internal/Magento/Framework/View/Page/Config/Structure.php]: Please remove 2nd space after `??`

**Review by ihor-sviziev — CHANGES_REQUESTED** [review]

**Comment by GraysonChiang** [comment]
I fixed.

**Review by ihor-sviziev — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @ihor-sviziev, thank you for the review.
[ENGCOM-2366](https://jira.corp.magento.com/browse/ENGCOM-2366) has been created to process this Pull Request

**Comment by magento-engcom-team** [comment]
Hi @GraysonChiang. Thank you for your contribution. 
We will aim to release these changes as part of 2.2.6. 
Please check the release notes for final confirmation.

Please, consider to port this solution to 2.3 release line.
You may use [Porting tool](https://github.com/magento/magento2/wiki/Simple-way-to-port-your-solutions-automatically) to port commits automatically.

**Pull request merged**